docs/consume-events.md

```diff
@@ -38,122 +38,193 @@ Once the authentication and authorization steps have been completed, you will re
 * 9098 (for access from AWS)
 * 9198 (for access from outside of AWS)
 
-### Develop and deploy Your consumer application
+### Develop and deploy your consumer application
 
 Many programming languages and frameworks offer libraries designed to interact with Kafka. To ensure full compatibility with the Event Bus, your code needs to authenticate with the AWS MSK cluster using the assigned role provided during the onboarding process. Additionally, consumers should reference the Confluent Schema Registry and use the appropriate schema to deserialize messages in Avro.
 
-See for instance this Java code that consumes messages from a topic named “test”:
+#### Client properties
+
+To connect to the Event Bus, consumers in **all programming languages** will need to set these properties, which are required unless otherwise specified:
+
+| Property | Value | Description | Notes |
+| --- | --- | --- | --- |
+| [bootstrap.servers](https://kafka.apache.org/documentation/#consumerconfigs_bootstrap.servers) | List of one or more Event Bus brokers. This will vary depending on the environment (dev, prod, etc.). | A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | Only one of the Event Bus clusters needs to be included in this list, but including more than one will ensure the application can start up if one of the Kafka servers is down. |
```

Comment:
Ah, I changed `cluster` to `broker` in the other page but missed it here. Good catch!
```suggestion
| [bootstrap.servers](https://kafka.apache.org/documentation/#consumerconfigs_bootstrap.servers) | List of one or more Event Bus brokers. This will vary depending on the environment (dev, prod, etc.). | A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. | Only one of the Event Bus brokers needs to be included in this list, but including more than one will ensure the application can start up if one of the Kafka servers is down. |
```